ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Components/NearbyTransitStopListItem.swift

```diff
@@ -0,0 +1,129 @@
+import SwiftUI
+import Swappable
+import MaasComponents
+
+public struct NearbyTransitStopListItem: View, Swappable {
+
+    @Environment(\.textStyleTextL) var textStyleTextL
+    @Environment(\.textStyleTextM) var textStyleTextM
+    @Environment(\.grayScale) var grayScale
+
+    public struct InputType {
+        public let data: StopWithSchedulesWithDepartures
+    }
+    public let input: InputType
+
+    public init(input: InputType) {
+        self.input = input
+    }
+
+    public var defaultBody: some View {
+        Cell(
+            content: { content },
+            prefix: { prefix }
+        )
+    }
+}
+
+// MARK: - Content
+
+private extension NearbyTransitStopListItem {
+
+    var content: some View {
+        VStack(alignment: .leading, spacing: Spacing.xs.value) {
+
+            VStack(alignment: .leading, spacing: Spacing.xxs.value) {
```

Comment:
Spacing.xxs.value could come from KMP constants 🤔

Reply:
`Spacing.xs.value` defines spacing for all content inside stack `Spacing.xxs.value` only for `stopName`, `stopDurationAndDirections`. We're doing this because our text lineHeight works not as aspected. 

So we could expose few constants in KMP like: `contentSpacing`, `textSpacing`.